test/system/500-networking.bats

```diff
@@ -799,4 +799,35 @@ EOF
     assert "$output" =~ "eth0"
 }
 
+@test "podman inspect list networks " {
+    run_podman create $IMAGE
+    cid=${output}
+    run_podman inspect --format '{{ .NetworkSettings.Networks }}' $cid
+    if is_rootless; then
+	is "$output" "map\[slirp4netns:.*" "NeworkSettings should contain one network named slirp4netns"
+    else
+	is "$output" "map\[podman:.*" "NeworkSettings should contain one network named podman"
+    fi
+    run_podman rm $cid
+
+    for network in "host" "none"; do
+        run_podman create --network=$network $IMAGE
+	cid=${output}
+	run_podman inspect --format '{{ .NetworkSettings.Networks }}' $cid
+	is "$output" "map\[$network:.*" "NeworkSettincs should contain one network named $network"
+	run_podman rm $cid
+    done
+
+    # Check with ns:/PATH
+    if ! is_rootless; then
+	netns=netns$(random_string)
+	ip netns add $netns
+	run_podman create --network=ns:/var/run/netns/$netns $IMAGE
+	cid=${output}
+	run_podman inspect --format '{{ .NetworkSettings.Networks }}' $cid
+	is "$output" "map\[ns:/var/run/netns/$netns.*" "NeworkSettings should contain path to netns"
+    fi
+    run_podman rm $cid
```

Comment:
again this is not correct both container: and ns: should not be added to the map, they are not a network!
If we want to match docker we should do so correctly.

Reply:
With this change podman and docker return the same content.
```
# ip netns add  dan
# podman create --network=ns:/var/run/netns/dan alpine
e0ae397eb613e6baf5994c35b6e15ff15768e6c7592a2e2cb63e1f8eff8f9a3e
# docker create --network=ns:/var/run/netns/dan alpine
d064be722af294c40631ec3ed1a07128af2805a8649b1f62282b966b094c051f
# ./bin/podman inspect --format '{{ .NetworkSettings.Networks }}' e0ae397eb613e6baf5994c35b6e15ff15768e6c7592a2e2cb63e1f8eff8f9a3e
map[ns:/var/run/netns/dan:0xc00089e000]
# docker inspect --format '{{ .NetworkSettings.Networks }}' d064be722af294c40631ec3ed1a07128af2805a8649b1f62282b966b094c051f
map[ns:/var/run/netns/dan:0xc00036c000]
```

Reply:
What docker version are you using. I don't see that? Are you sure your docker is not linked to podman?

AFAIK docker does not even support the `ns:` syntax only the `contianer:` one, at least on my version 20.10.23

Reply:
# docker create --network=ns:/var/run/netns/dan alpine
67ba122e06560ada92dc8fffa7cd0686af46500c11e424908c6c68003ea2236a
# docker inspect --format '{{ .NetworkSettings.Networks }}' 67ba122e06560ada92dc8fffa7cd0686af46500c11e424908c6c68003ea2236a
map[ns:/var/run/netns/dan:0xc0004b7980]
# docker -v
Docker version 20.10.22, build 3a2c30b

Reply:
20.10.23 is not available in docker-ce, as far as I see.

Reply:
docker -v is showing you the client information, try docker version/info.

Reply:
```
 docker version
Client: Docker Engine - Community
 Version:           20.10.22
 API version:       1.41
 Go version:        go1.18.9
 Git commit:        3a2c30b
 Built:             Thu Dec 15 22:28:45 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.22
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.9
  Git commit:       42c8b31
  Built:            Thu Dec 15 22:26:25 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.15
  GitCommit:        5b842e528e99d4d4c1686467debf2bd4b88ecd86
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Reply:
I am not sure why you want to hide this information from the users.  Seems like a good thing for users to be able to see which networks their container is attached to.

Reply:
It is not hidden, all info is already in the basic struct.
The map you are adding uses the network name as key. Neither ns: nor contianer: is considered a network, they are a network mode.

Reply:
I am confused because https://docs.docker.com/engine/reference/run/#network-settings does not mention the ns: key and AFAIK docker does not support it.